src/ecdsa.js

```diff
@@ -223,6 +223,5 @@ module.exports = {
   deterministicGenerateK: deterministicGenerateK,
   recoverPubKey: recoverPubKey,
   sign: sign,
-  verify: verify,
-  verifyRaw: verifyRaw
```

Comment:
Already noted as breaking in 1.5.x branch,  and this is 2.0.0 Dev.
On 4 May 2015 3:09 am, "Wei Lu" <notifications@github.com> wrote:

> In src/ecdsa.js
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/403#discussion_r29557915>
> :
>
> > @@ -223,6 +223,5 @@ module.exports = {
> >    deterministicGenerateK: deterministicGenerateK,
> >    recoverPubKey: recoverPubKey,
> >    sign: sign,
> > -  verify: verify,
> > -  verifyRaw: verifyRaw
>
> Note: major version bump
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/403/files#r29557915>.
>